### ISSUE TITLE
Audio Streaming Failed 

### ISSUE THREAD

**Issue by lostpuma** [HermesApp/Hermes#212]
I am no longer able to connect to Pandora through Hermes, even after multiple restarts and reconnect attempts.

It shows the following error:

"An error has occured with Pandora.

Try Again

Audio streaming failed."

Is Pandora now blocking this application or is this an isolated incident?

**Comment by nicolas3470** [comment]
I'm also seeing this. Tried removing and re-installing, but no dice.

**Comment by brentmc79** [comment]
Same here. Streaming has failed for the past two days.

**Comment by lostpuma** [comment]
I went into the settings and under the Network tab, I enabled "Use proxy for audio" and it is working again.  I hope that resolves the issue for you guys as well.

**Comment by nriley** [comment]
I can't reproduce this unfortunately, but hope the proxy setting helps others.

**Comment by nicolas3470** [comment]
Proxying audio did not work. Signing out, then signing back in has me stuck on the choose a station screen. Pulling out the drawer via cmd+shift+s shows no stations.

**Comment by winny-** [comment]
Unfortunately Hermes' audio streaming code is a bit unreliable at the moment. I hope to replace it with Cocoa native tools, or try to integrate with VLCKit.

**Comment by DrPat** [comment]
Pandora has blocked my app now it seems on my account from signing in after the latest bug issue when I signed out and then tried to sign back in.

**Comment by DrPat** [comment]
Macbook 10.10.2 

Still having streaming problems. After one play the error window that says try again comes up and play stops. Trying different stations sometimes will get another song to play but only one song plays now before streaming stops. I can eventually get another station to play but still only one song before streaming stops. I am somewhat new to Hermes platform have version 1.2.5. This has been happening since i fixed the last issue of not being able to log back on. For now it is one song at a time. It is amazing to have such extremely smart people working on this project hoping for a fix. -DrPat

**Comment by skywinder** [comment]
Same problem as @lostpuma.
Usually I use "Use proxy for audio" checked. 
I use proxy. But it stop works spontaneously.
If I uncheck it and check it again - it start work again.

**Comment by phallstrom** [comment]
Checking 'Use proxy for audio' fixed it for me (for now anyway).

**Comment by DrPat** [comment]
The audio stream interruption and error causing loss of the connection to Pandora. I noticed that in the windows 7 gadget pandora interface every so often if you do not use the like or dislike buttons a pull down window appears asking if you are still listening and if you do not say yes then the stream stops until you click on the still listening button. My question is how does hermes handle this and is the Pandora stream have something that checks on continuous streaming that they think is not being monitored. It is a simple non coder question which could be answered yes or no but i have noticed the question that streaming failures could coincide with continuous listening time limits possibly. Thank you for all your hard work but possibly fun for you coders it pays off for me this is the best way to listen to music keep it up. - Pat

**Comment by winny-** [comment]
No, I don't believe the unreliable streaming is a consequence of this "Still Listening" functionality.

From my understanding, the "Still Listening" dialog is the client

1. Checking if the UI has been interacted with since *n* minutes ago, (I think at least 120 min, but I have no concrete data),
2. Pausing the Playback and asking if you're still listening if (1) is true.

**Comment by DrPat** [comment]
Interesting thanks for taking the time to answer.. 

on the problem.. I have now noticed to keep it going when streaming stops I am able to switch to another station and it will play that station and after I play that station for a time then I can go back to the other station and it plays again. And for me it could be after 120 mins have not timed it though at this time. Keep up the good work it is the best and most convent way of listening to internet radio... the issue is only a bother at most....

**Comment by cdpritcha** [comment]
Checking "Use proxy for audio" fixed the issue for me.
